### PR TITLE
fix: add rest source, rest processor

### PR DESCRIPTION
**Pull request by AntonNemtsev**

### CODE REVIEW

File: blocks/rest/events.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                 url: str,
+                 event: Type[BaseModel],
+                 timeout: float = 5.0) -> None:
+        self.url = url
+        self.event = event
+        self.timeout = timeout
+
+
+class OutputUrl:
+    def __init__(self,
```
Comment (tofuurem) [inline blocks/rest/events.py:17]: please use the formatting used throughout the project.

like:
```python3
def __init__(
    url: str,
    ...
) -> None:
```

File: blocks/rest/events.py [hunk 2]
```diff
@@ -0,0 +1,27 @@
+from typing import Type, Optional
+
+from pydantic import BaseModel
+
+
+class InputUrl:
```
Comment (tofuurem) [inline blocks/rest/events.py:6]: What's the reason not to use dataclass or pydantic here, but a pure object?

File: blocks/rest/sources.py [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+                data = response.json()
+                for obj in data:
+                    try:
+                        events.append(parse_obj_as(endpoint.event, obj))
+                    except Exception:
+                        if not self.ignore_errors:
+                            raise
+            except requests.RequestException:
+                if not self.ignore_errors:
+                    raise
```
Comment (tofuurem) [inline blocks/rest/sources.py:72]: and what's that blank `raise'?

Comment (tribunsky-kir) [reply blocks/rest/sources.py:72]: This will actually re-raise requests' exception up, that's okay.

**Review by tofuurem — APPROVED** [review]

**Pull request merged**